src/Common/mysqlxx/tests/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 clickhouse_add_executable (mysqlxx_pool_test mysqlxx_pool_test.cpp)
-target_link_libraries (mysqlxx_pool_test PRIVATE mysqlxx dbms clickhouse_common_config loggers_no_text_log)
+target_link_libraries (mysqlxx_pool_test PRIVATE mysqlxx clickhouse_common_config loggers_no_text_log)
```

Comment:
SO finally `dbms` is not needed? Not even the `02443_detach_attach_partition`?

Reply:
For this target - yes, not sure that I got part about `02443_detach_attach_partition`